internal/net/grpc/client.go

```diff
@@ -39,34 +40,35 @@ type DialOption = grpc.DialOption
 
 type Client interface {
 	StartConnectionMonitor(ctx context.Context) (<-chan error, error)
-	Connect(ctx context.Context, addr string) error
+	Connect(ctx context.Context, addr string, gopts ...DialOption) error
 	Disconnect(addr string) error
 	Range(ctx context.Context,
 		f func(addr string,
-			conn *grpc.ClientConn,
-			copts ...grpc.CallOption) error) error
+			conn *ClientConn,
+			copts ...CallOption) error) error
 	RangeConcurrent(ctx context.Context,
 		concurrency int,
 		f func(addr string,
-			conn *grpc.ClientConn,
-			copts ...grpc.CallOption) error) error
+			conn *ClientConn,
+			copts ...CallOption) error) error
 	Do(ctx context.Context,
-		addr string, f func(conn *grpc.ClientConn,
-			copts ...grpc.CallOption) (interface{}, error)) (interface{}, error)
+		addr string, f func(conn *ClientConn,
+			copts ...CallOption) (interface{}, error)) (interface{}, error)
 	GetAddrs() ([]string, []string)
-	GetDialOption() []grpc.DialOption
-	GetCallOption() []grpc.CallOption
+	GetDialOption() []DialOption
+	GetCallOption() []CallOption
 	Close() error
 }
 
 type gRPCClient struct {
-	addrs []string
-	conns gRPCConns
-	hcDur time.Duration
-	gopts []grpc.DialOption
-	copts []grpc.CallOption
-	eg    errgroup.Group
-	bo    backoff.Backoff
+	addrs       []string
+	clientCount uint64
+	conns       gRPCConns
+	hcDur       time.Duration
+	gopts       []DialOption
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
I prefer `dopts` for `DialOption` that clarifies its meaning.